content/docs/2.11/scalers/cpu.md

```diff
@@ -8,7 +8,7 @@ go_file = "cpu_memory_scaler"
 
 > **Notice:**
 > - This scaler **requires prerequisites**. See the 'Prerequisites' section.
-> - This scaler will never scale to 0 and even when user defines multiple scaler types (eg. Kafka + cpu/memory, or Prometheus + cpu/memory), the deployment will never scale to 0.
+> - This scaler can scale to 0 only when user defines at least one additional scaler which is not CPU or Memory (eg. Kafka + CPU/Memory, or Prometheus + CPU/Memory) and sets minReplicaCount to 0.
```

Comment:
```suggestion
> - This scaler can scale to 0 only when user defines at least one additional scaler which is not CPU or Memory (eg. Kafka + CPU, or Prometheus + CPU) and sets minReplicaCount to 0.
```

Reply:
```suggestion
> - This scaler can scale to 0 only when user defines at least one additional scaler which is not CPU or Memory (eg. Kafka + CPU, or Prometheus + CPU) and minReplicaCount is 0.
```
what about not using the word `set` also, since minReplicaCount is by default zero therefore it doesnt have to be `set` necessarily?

Reply:
Makes sense. Also annotate rename minReplicaCount to `minReplicaCount` please